apps/src/codebridge/PopUpButton/pop-up-button.module.scss

```diff
@@ -27,5 +27,5 @@
 // We need !important because of css specificity- this would be overriden
 // by display: none otherwise.
 button.active {
-  display: inherit !important;
```

Comment:
Just switched the approach so button is still 'findable' in hidden state so it can be focused and then made visible again. This is effectively the same thing